addons/arsenal/functions/fnc_buttonHide.sqf

```diff
@@ -0,0 +1,52 @@
+/*
+ * Author: Alganthe
+ * Hide / show arsenal interface.
+ *
+ * Arguments:
+ * 0: Arsenal display <DISPLAY>
+ *
+ * Return Value:
+ * None
+ *
+ * Public: No
+*/
+#include "script_component.hpp"
+#include "..\defines.hpp"
+
+params ["_display"];
+
+private _showToggle = !ctrlShown (_display displayCtrl IDC_menuBar);
+
+if (_showToggle) then {
+    [_display, _display displayCtrl GVAR(currentLeftPanel)] call FUNC(populatePanel)
+};
+
+{
+    private _ctrl = _display displayctrl _x;
+    _ctrl ctrlshow _showToggle;
+    _ctrl ctrlcommit 0.15;
+} foreach [
```

Comment:
`forEach`... is debatable though.

Reply:
`count` might actually be beneficial here. I've seen this function while profiling. Might aswell have been the call to populatePanel in L21 though

Reply:
Very likely to be the fillPanel func being called.